lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -560,12 +560,16 @@ llvm::Error Caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     NodeValue in;
     ASSIGN_VALUE_OR_RETURN_ERR(in,
                                getNodeValueOrCreateConstantByName(op.input(0)));
-    if (in.getType()->dims().size() > 2) {
-      size_t axis = 1;
-      if (dict.count("axis")) {
-        ASSIGN_VALUE_OR_RETURN_ERR(axis, loadInt(dict["axis"]));
-      }
 
+    auto originalInputDims = in.getType()->dims();
```

Comment:
Can you document the expected behavior when flattening happens and how the resulted shape after FC should look like.